React/Modules/RCTDevMenu.m

```diff
@@ -185,7 +185,7 @@ - (instancetype)init
       [weakSelf.bridge.eventDispatcher sendDeviceEventWithName:@"toggleElementInspector" body:nil];
     }]];
 
-    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"Chrome";
+    _webSocketExecutorName = [_defaults objectForKey:@"websocket-executor-name"] ?: @"JS Remotely";
```

Comment:
Is anyone working on a debugger that talks directly to JSC? That would be awesome!